gen/dcos-config.yaml

```diff
@@ -385,6 +385,7 @@ package:
       MESOS_FAIR_SHARING_EXCLUDED_RESOURCE_NAMES={{ fair_sharing_excluded_resource_names }}
       MESOS_FILTER_GPU_RESOURCES={{ gpus_are_scarce }}
       GLOG_drop_log_memory=false
+      LIBPROCESS_WORKER_THREADS=16
```

Comment:
Shouldn't this be `LIBPROCESS_NUM_WORKER_THREADS`?
Do we also want to bump https://github.com/dcos/dcos/blob/1.9.5/gen/dcos-config.yaml#L455 ?

Reply:
Aah, thanks for the catch, @adam-mesos . Just fixed the typo.

About the change to executor value, I am not sure if that's needed. We need it for master because there are several zookeeker groups in master but the same is not true for agents/executors. Any thoughts?